recipe/meta.yaml

```diff
@@ -42,7 +37,8 @@ outputs:
         - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
         - numpy                                  # [build_platform != target_platform]
-        - pytorch ={{ pytorch_version }}={{ torch_proc_type }}*      # [build_platform != target_platform]
+        - pytorch                                # [build_platform != target_platform]
+        - pytorch =*={{ torch_proc_type }}*      # [build_platform != target_platform]
```

Comment:
Why both?

Reply:
so that it picks up the cuda/cpu difference.